libraries/AP_BattMonitor/AP_BattMonitor_INA2xx.cpp

```diff
@@ -385,16 +429,24 @@ void AP_BattMonitor_INA2XX::timer(void)
         if (!read_word16(REG_231_SHUNT_VOLTAGE, bus_voltage16) ||
             !read_word16(REG_231_CURRENT, current16)) {
             failed_reads++;
-            if (failed_reads > 10) {
-                // device has disconnected, we need to reconfigure it
-                dev_type = DevType::UNKNOWN;
-            }
             return;
         }
         voltage = bus_voltage16 * voltage_LSB;
         current = current16 * current_LSB;
         break;
     }
+
+    case DevType::INA260: {
+        int16_t current16, bus_voltage16;
```

Comment:
We don't allow stacking variable declarations like this in our style guide.

... it's just more of the same in this file, 'though...